tensorboard/backend/event_processing/data_provider.py

```diff
@@ -19,11 +19,15 @@
 from __future__ import print_function
 
 import six
+import collections
+import json
+import base64
```

Comment:
nit: Standard library imports (these three new ones) go in a separate
import section than third-party imports (`six`), and each section should
be alphabetized:
<https://google.github.io/styleguide/pyguide.html#313-imports-formatting>